python/pyspark/testing/connectutils.py

```diff
@@ -184,6 +184,7 @@ def setUpClass(cls):
             .remote(cls.master())
             .getOrCreate()
         )
+        cls._legacy_sc = PySparkSession._instantiatedSession._sc
```

Comment:
refer to `test_connect_basic`

https://github.com/apache/spark/blob/714feb7b5aec948c3499ad486dc63ea89a241ffa/python/pyspark/sql/tests/connect/test_connect_basic.py#L66-L68